src/Sulu/Bundle/AdminBundle/Resources/js/components/AutoComplete/index.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+import AutoComplete from './AutoComplete';
+import Suggestion from './Suggestion';
+
+export {
+    AutoComplete,
+    Suggestion,
```

Comment:
I think we've got a new structure that also applies here 😃 

Please add `Suggestion` as static property to `AutoComplete` and only export `AutoComplete` as default.